website/content/configuration/calico.md

```diff
@@ -7,9 +7,11 @@ weight: 20
 
 As of Calico 3.18 (from early 2021), Calico now supports limited
 integration with MetalLB. Calico can be configured to announce
-LoadBalancer IPs itself via BGP. Simply run MetalLB *without* the
-speaker pods, and then configure the IP range(s) that you want Calico
-to announce.
+LoadBalancer IPs itself via BGP. Simply run MetalLB, apply an `IPAddressPool` *without* any
```

Comment:
Calico can be configured to announce the LoadBalancer IPs via BGP.

Reply:
Fixed